itou/www/companies_views/views.py

```diff
@@ -111,17 +111,30 @@ def job_description_list(request, template_name="siaes/job_description_list.html
                 ).first():
                     job_description.is_active = is_active
                     job_description.save(update_fields=["is_active"])
-                    messages.success(request, f"Le recrutement est maintenant {'ouvert' if is_active else 'fermé'}.")
+                    messages.success(
+                        request,
+                        f"Le recrutement est maintenant {'ouvert' if is_active else 'fermé'}.",
+                        extra_tags="toast",
+                    )
                 else:
                     messages.error(request, "La fiche de poste que vous souhaitiez modifier n'existe plus.")
             case "block_job_applications":
                 siae = form.save()
-                messages.success(
-                    request,
-                    "La réception de candidature est temporairement bloquée."
-                    if siae.block_job_applications
-                    else "La structure peut recevoir des candidatures.",
-                )
+                if siae.block_job_applications:
+                    messages.warning(
+                        request,
+                        (
+                            "La réception de candidature est temporairement bloquée.||"
+                            "Pour recevoir de nouvelles candidatures, veuillez désactiver le blocage"
```

Comment:
Le "veuillez" donne l'impression qu'on demande à la personne de le faire alors qu'on veux juste lui indiquer ce qu'il faut faire maintenant qu'elle a bloqué les candidatures, _my 2ct_.
```suggestion
                            "Pour recevoir de nouvelles candidatures, désactiver le blocage"
```